src/ui/src/wds/WdsJsonViewer.utils.ts

```diff
@@ -0,0 +1,47 @@
+export type JSONValue = string | number | boolean | null;
```

Comment:
I think I'd just pack all these utils inside `WdsJsonViewer.vue` and export them from there, since they're specific to `WdsJsonViewer`, what do you think?